source/Octopus.Server.Client/Octopus.Server.Client.csproj

```diff
@@ -7,6 +7,8 @@
     <AssemblyName>Octopus.Server.Client</AssemblyName>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
     <Authors>Octopus Deploy</Authors>
+    <PackageProjectUrl>https://github.com/OctopusDeploy/Nevermore</PackageProjectUrl>
```

Comment:
Should this point to OctopusClients rather than Nevermore?

Reply:
🙈 fixed!